include/filesystem.h

```diff
@@ -14,6 +16,21 @@ extern "C" {
 
 void filesystem_init();
 
+static inline const char *process_path(const char *path, const char *dev_name) {
+    unsigned int dev_name_len = strlen(dev_name);
+    const char *device_path = path;
+    while (*device_path == '/') {
+        // Skip leading space
```

Comment:
misleading comment: '/' is not a space, or am I missing something?
... and how about other characters at the beginning of the path? What else should we skip/cope with?

Reply:
Thanks, it's a "typo", should be "Skip leading slashes"